docs/pages/consent/subscribe-broadcast.mdx

```diff
@@ -0,0 +1,189 @@
+
+The message will display in the subscriber’s main inbox because clicking the Subscribe button not only creates the subscription, it also approves contact from the broadcaster.
+
+### Define the broadcast
+
+With the example subscribe-broadcast page running, go to [http://localhost:3000/subscribe/button](http://localhost:3000/subscribe/button). 
+
+You should see a dropdown list of broadcasts and a **Subscribe** button.
+
+![subscribe-broadcast web page](https://raw.githubusercontent.com/xmtp/docs-xmtp-org/main/docs/pages/img/subscribe-broadcast-page.png)
+
+To edit the available broadcasts, in the **subscribe-broadcast** repo, see the `src/app/broadcastConfigs.ts` file.
+
+```tsx
+interface BroadcastConfig {
+  name: string;
+  address: string;
+}
+
+export const broadcastConfigs: BroadcastConfig[] = [
+  {
+    address: "0x62EEd858af7590fbCaE803d208c53ddBb0D1309c",
+    name: "Hello World Wild Web",
+  },
+];
+```
+
+In the **address** field, enter a string that is the raw 0x Ethereum address of the broadcast sender address.
+
+In the **name** field, enter a string that is the name of the broadcast.
+
+### Define the example welcome broadcast message
+
+When you connect a wallet to the subscribe-broadcast page and click **Subscribe**, the page will send an example welcome message to the connected wallet address on the XMTP `dev` network.
+
+The example welcome message is defined in `src/lib/broadcasterConfigs.ts` in the **broadcast-example-api** repo.
+
+```tsx
+const broadcastConfigs: BroadcastConfig[] = [
+  {
+    address: "0x62EEd858af7590fbCaE803d208c53ddBb0D1309c",
+    greeting: "Welcome to the World Wild Web!",
+    id: "XMTP",
+  },
+];
+```
+
+In the **address** field, enter a string that is the raw 0x Ethereum address of the broadcast sender address.
+
+In the **greeting** field, enter a string that is the welcome message you want to send when someone subscribes to your broadcast.
+
+In the **id** field, enter a string that is a unique ID for the broadcast.
+
+### Send and receive the example welcome broadcast message
+
+1. With the example subscribe-broadcast page running, go to [http://localhost:3000/subscribe/button](http://localhost:3000/subscribe/button). 
+2. Connect the wallet you want to subscribe to the broadcast. To demonstrate the consent feature of the Subscribe button, be sure to connect a wallet address that has not already exchanged messages with the broadcast address.
+3. Select the broadcast you defined.
+4. Click **Subscribe**.
+5. Go to XXXXXXxmtp web app on devXXXXXX and connect the subscribed wallet address. The example welcome broadcast message displays in the main inbox.
+
+## Send an example broadcast message
+
+You can also use the example subscribe-broadcast page to send an example broadcast message.
+
+1. In the **broadcast-example-api** repo, update `addresses.json`  to include only the address you just subscribed to the broadcast.
```

Comment:
In a real-life flow, how does the connected address that clicked the Subscribe button get saved so that this manual step isn't required?

And more specifically, how can a creator get their subscriber list without having to rely on the third-party service they may be using to actually do their subscribe/broadcast? I think this needs to be possible when we say that creators can take their audiences with them.